Auth container does not build with new backend code

It's amazing that the backend has been released, but when I run `build.sh` I get the following error:

```
[+] Building 94.3s (16/17)                                                                                                                                                                                                     docker:default
 => [internal] load build definition from Dockerfile                                                                                                                                                                                     0.0s
 => => transferring dockerfile: 372B                                                                                                                                                                                                     0.0s
 => [internal] load metadata for docker.io/library/alpine:latest                                                                                                                                                                         0.2s
 => [internal] load metadata for docker.io/library/golang:1.24-alpine                                                                                                                                                                    0.2s
 => [internal] load .dockerignore                                                                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                                          0.0s
 => [builder 1/8] FROM docker.io/library/golang:1.24-alpine@sha256:fc2cff6625f3c1c92e6c85938ac5bd09034ad0d4bc2dfb08278020b68540dbb5                                                                                                      0.0s
 => [internal] load build context                                                                                                                                                                                                        0.0s
 => => transferring context: 68.63kB                                                                                                                                                                                                     0.0s
 => [stage-1 1/4] FROM docker.io/library/alpine:latest@sha256:4bcff63911fcb4448bd4fdacec207030997caf25e9bea4045fa6c8c44de311d1                                                                                                           0.0s
 => CACHED [stage-1 2/4] WORKDIR /root/                                                                                                                                                                                                  0.0s
 => CACHED [stage-1 3/4] RUN apk add --no-cache tzdata                                                                                                                                                                                   0.0s
 => CACHED [builder 2/8] WORKDIR /app                                                                                                                                                                                                    0.0s
 => CACHED [builder 3/8] COPY auth/ ./auth                                                                                                                                                                                               0.0s
 => CACHED [builder 4/8] COPY proto/ ./proto                                                                                                                                                                                             0.0s
 => CACHED [builder 5/8] COPY util/ ./util                                                                                                                                                                                               0.0s
 => CACHED [builder 6/8] COPY mongoutil/ ./mongoutil                                                                                                                                                                                     0.0s
 => CACHED [builder 7/8] WORKDIR /app/auth                                                                                                                                                                                               0.0s
 => ERROR [builder 8/8] RUN go build -o auth cmd/auth/auth.go                                                                                                                                                                           94.1s
------                                                                                                                                                                                                                                        
 > [builder 8/8] RUN go build -o auth cmd/auth/auth.go:                                                                                                                                                                                       
0.456 go: downloading github.com/golang-jwt/jwt/v5 v5.2.2                                                                                                                                                                                     
0.456 go: downloading github.com/gofiber/contrib/jwt v1.1.2                                                                                                                                                                                   
0.456 go: downloading github.com/getsentry/sentry-go v0.34.1                                                                                                                                                                                  
0.457 go: downloading github.com/gofiber/fiber/v2 v2.52.8                                                                                                                                                                                     
0.459 go: downloading github.com/google/uuid v1.6.0
0.461 go: downloading github.com/joho/godotenv v1.5.1
0.957 go: downloading github.com/matthewhartstonge/argon2 v1.3.1
1.071 go: downloading github.com/segmentio/kafka-go v0.4.48
1.300 go: downloading go.mongodb.org/mongo-driver v1.17.3
3.255 go: downloading google.golang.org/grpc v1.72.1
3.405 go: downloading github.com/MicahParks/keyfunc/v2 v2.1.0
3.429 go: downloading golang.org/x/sys v0.33.0
4.410 go: downloading golang.org/x/crypto v0.38.0
5.195 go: downloading github.com/mattn/go-colorable v0.1.14
5.224 go: downloading github.com/mattn/go-isatty v0.0.20
5.258 go: downloading github.com/mattn/go-runewidth v0.0.16
5.282 go: downloading github.com/valyala/bytebufferpool v1.0.0
5.320 go: downloading github.com/valyala/fasthttp v1.62.0
5.564 go: downloading google.golang.org/protobuf v1.36.6
5.612 go: downloading golang.org/x/text v0.25.0
5.640 go: downloading golang.org/x/net v0.40.0
5.650 go: downloading github.com/rivo/uniseg v0.4.7
5.921 go: downloading github.com/youmark/pkcs8 v0.0.0-20240726163527-a2c0da244d78
5.946 go: downloading google.golang.org/genproto/googleapis/rpc v0.0.0-20250519155744-55703ea1f237
6.527 go: downloading github.com/klauspost/compress v1.18.0
6.697 go: downloading github.com/pierrec/lz4/v4 v4.1.15
7.178 go: downloading github.com/andybalholm/brotli v1.1.1
9.209 go: downloading github.com/xdg-go/scram v1.1.2
9.283 go: downloading github.com/xdg-go/stringprep v1.0.4
9.383 go: downloading github.com/golang/snappy v0.0.4
9.551 go: downloading github.com/montanaflynn/stats v0.7.1
9.749 go: downloading github.com/xdg-go/pbkdf2 v1.0.0
9.834 go: downloading golang.org/x/sync v0.14.0
93.59 # perfice.adoe.dev/auth/internal
93.59 internal/session.go:44:16: undefined: jwt
93.59 internal/session.go:155:12: undefined: jwt
93.59 internal/session.go:161:11: undefined: jwt
------
Dockerfile:10
--------------------
   8 |     
   9 |     WORKDIR /app/auth
  10 | >>> RUN go build -o auth cmd/auth/auth.go
  11 |     
  12 |     FROM alpine:latest
--------------------
ERROR: failed to build: failed to solve: process "/bin/sh -c go build -o auth cmd/auth/auth.go" did not complete successfully: exit code: 1

```

Adding the go JWT library to `auth/internal/sessions.go` fixes this

--- a/server/auth/internal/session.go
+++ b/server/auth/internal/session.go
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	"github.com/google/uuid"
+    "github.com/golang-jwt/jwt/v5"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"perfice.adoe.dev/mongoutil"

